migrations/V50__Add_Operational_Solution_Subtask.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TYPE OPERATIONAL_SOLUTION_SUBTASK_STATUS AS ENUM (
+    'TODO',
+    'IN_PROGRESS',
+    'DONE'
+);
+
+CREATE TABLE operational_solution_subtasks (
+    id UUID PRIMARY KEY NOT NULL,
+    solution_id UUID NOT NULL,   --foreign key to model plan
+    name ZERO_STRING,
+    status OPERATIONAL_SOLUTION_SUBTASK_STATUS NOT NULL DEFAULT 'TODO',
+
+    --META
+    created_by UUID NOT NULL REFERENCES public.user_account (id) MATCH SIMPLE,
+    created_dts TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    modified_by UUID REFERENCES public.user_account (id) MATCH SIMPLE,
+    modified_dts TIMESTAMP WITH TIME ZONE
+);
+
+
+ALTER TABLE operational_solution_subtasks
+ADD CONSTRAINT fk_operational_solution_subtasks FOREIGN KEY (solution_id)
+REFERENCES public.operational_solution (id) MATCH SIMPLE
+ON UPDATE NO ACTION
+ON DELETE NO ACTION
```

Comment:
One other thing I realized is that auditing should be turned on for the table. Migration V23 is a good example how to turn that on. One caveat is that migration is using a function that is not updated now to include if the table uses the user table (just need to set that in the audit.table_config for this entry).